string desired_frame
string relative_to_frame

---

float64 x
float64 y
float64 theta
bool result
